docs/sources/_index.md

```diff
@@ -38,6 +41,18 @@ Beyla offers the following features:
 - listen to the Kubernetes API to decorate metrics and traces with Pods and Services metadata
 - simple setup for Grafana customers already using Grafana Alloy
 
+## eBPF overview
+
+eBPF stands for Extended Berkeley Packet Filter, and allows attaching applications to different points of the Linux Kernel. eBPF applications run in privileged mode and allow the runtime information of the Linux Kernel to be inspected: system calls, network stack, as well as inserting probes in user space applications.
```

Comment:
> eBPF stands for Extended Berkeley Packet Filter

This is not true anymore. From some time ago, eBPF stopped being an acronym and just means "eBPF".

Reply:
Yeah I agree, we can just remove `stands for Extended Berkeley Packet Filter, and`.